### ISSUE TITLE
Map is outdated

### ISSUE THREAD

**Issue by DoubleWhisky** [ogruetzmann/ModeS#11]
Small update required. Current coverage/usage map makes some of us confusing from time to time.

Thanks in advance
M

**Comment by ogruetzmann** [comment]
You are absolutely right. I'm not a genius in this area, but I'll try to make something more meaningful. Can't promise it'll look good... 😬

**Comment by droesen** [comment]
FYI, at the moment the "Mode S Map" link is broken, "Not found"...

**Comment by ogruetzmann** [comment]
Seems like it's about time to find someone who is able to create such a map :-S

**Comment by droesen** [comment]
Public domain (no license required, no attribution needed) GIS data: https://www.naturalearthdata.com/
Tool to create visualization from that: https://www.qgis.org/
Source for ICAO FIR outlines in QGIS-supported format: no idea
Time and nerves to get it done: none, yet. :-P

**Comment by pierr3** [comment]
This should do it :) 

Map background: (c) Eurocontrol 2019

![](https://i.imgur.com/0VUhMad.png)

**Comment by ogruetzmann** [comment]
Hi Pierre, that one looks great. Do you know if it's allowed to be used with that background?

Am 3. September 2020 23:05:46 MESZ schrieb Pierre Ferran <notifications@github.com>:

**Comment by pierr3** [comment]
It's a bit 50/50, the copyright statement on the map says the following:

© 2019 The European Organisation for the Safety of Air Navigation (EUROCONTROL).
This document is published by EUROCONTROL for information purposes . It maybe
copied in whole or in part, provided that EUROCONTROL is mentioned as the source
and to the extent justified by the non-commercial use (not for sale). The information in
this document may not be modified without prior written permission from EUROCONTROL.

This is technically derivate work, with no modification of the source file. The way I see, we are using it non-commercially and this map was partly made using our tax money, hence I think we have a pretty good argument as long as we retain the above copyright notice and maybe disclaimer that this is solely for simulation use on VATSIM.
